tests/commonTemplates_test.go

```diff
@@ -208,6 +208,30 @@ var _ = Describe("Common templates", func() {
 	})
 
 	Context("resource change", func() {
+
+		Context("namespace change", func() {
+			AfterEach(func() {
+				strategy.RevertToOriginalSspCr()
+			})
+
+			It("[test_id:6057] should not fail to update commonTemplates.namespace", func() {
+				foundSsp := getSsp()
+				const testNamespace = "ssp-operator-functests"
```

Comment:
Can you create a new namespace for this test? Would be better than polluting the testing namespace.
Also the tests can be run in a different namespace, so instead of a constant, you can use `strategy.GetNamespace()`.